tests/app/questionnaire/routing/test_utils.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime, timezone
+
+import pytest
+from freezegun import freeze_time
+
+from app.questionnaire.routing.utils import parse_datetime
+
+
+@pytest.mark.parametrize(
+    "date_string, date_format",
+    [
+        ("2021-10", "%Y-%m"),
+        ("2021-10-29", "%Y-%m-%d"),
+        ("2021-10-29T10:53:41.511833+00:00", "%Y-%m-%dT%H:%M:%S.%f%z"),
```

Comment:
Maybe add the Zulu variant for testing?

Reply:
The current version of `parse_datetime` doesn't support Zulu or ISO without milliseconds. I have changed the implementation in https://github.com/ONSdigital/eq-questionnaire-runner/pull/, so will address there.